workflows/genomeassembly.nf

```diff
@@ -312,24 +312,27 @@ workflow GENOMEASSEMBLY {
                        GENOMESCOPE_MODEL.out.hist,
                        GENOMESCOPE_MODEL.out.ktab
         )
-    }
+    }*/
```

Comment:
Was it commented out with the purpose of debugging and does it need uncommenting?